src/main/java/com/epam/controller/ReservationController.java

```diff
@@ -0,0 +1,33 @@
+package com.epam.controller;
+
+import com.epam.model.Reservation;
+import com.epam.service.ReservationService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.ModelMap;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import java.security.Principal;
+import java.util.List;
+
+@Controller
+public class ReservationController {
+    @Autowired
+    ReservationService reservationService;
+
+    @GetMapping("/testadmin")
+    public String listReservations(Principal user, ModelMap modelMap) {
+        List<Reservation> list = reservationService.listReservations();
+        modelMap.addAttribute("listReservation", list);
+        return "testadmin";
+    }
+
+    @RequestMapping(value = "/delete/{id}", method = RequestMethod.GET)
```

Comment:
Why not to use Delete HTTP request method?

Reply:
URLs of the controllers should be unique. Do you think, that just only 'delete' will be here?

Reply:
I thought that we should use only get and post methods